### PR TITLE
Adding Venus for gender equality

### LINKED ISSUE #1: How about adding Venus as well?
Should be possible right?

### PR DESCRIPTION
**Pull request by petrisch**
Fixes #1

### CODE REVIEW

**Comment by petrisch** [comment]
Ok your approach is much better :-D

File: Venus [hunk 1]
```diff
@@ -0,0 +1 @@
+Venus is great also
```
Comment (mangott) [inline Venus:1]: also should be placed before great. Or write: "Venus is great too"

**Review by mangott — CHANGES_REQUESTED** [review]

**Review by mangott — APPROVED** [review]
Approved ;-)

File: Venus (hunk 1 above)
Comment (petrisch) [reply Venus]: should have rebased this, but I'm in a call and only have the github editor... 
Don't know how to do this here...

**Pull request merged**